dts/bindings/can/atmel,sam-can.yaml

```diff
@@ -0,0 +1,19 @@
+description: Specialization of Bosch m_can CAN-FD controller for Atmel SAM
+
+compatible: "atmel,sam-can"
+
+include: bosch,m-can.yaml
+
+properties:
+    peripheral-id:
+      type: int
+      required: true
+      description: peripheral ID
+    pinctrl-0:
+      type: phandles
+      required: true
+      description: |
+        PIO configuration for CAN_RX and CAN_TX. We expect that the phandles will
+        reference pinctrl nodes.  These nodes will have a nodelabel that
+        matches the Atmel SoC HAL defines and be of the form
+          pinctrl-0 = <&pa8a_uart_urxd &pa9a_uart_utxd>;
```

Comment:
This example should use can pins instead of uart.

Reply:
Whoops. cpy&paste :-)